config/vufind/Primo.ini

```
@@ -77,6 +80,12 @@ default_noresults_recommend[] = RemoveFilters
 ; the Primo API
 bulkSize = 20
 
+; Enable or disable display of CDI attributes (e.g. Retracted Publication).
+; Default is true.
+; See https://knowledge.exlibrisgroup.com/Primo/Content_Corner/Central_Discovery_Index/Documentation_and_Training/Documentation_and_Training_(English)/CDI_-_The_Central_Discovery_Index/CDI_Document_Attributes
+; for more information.
```

Minor edit to make extremely long line at least four characters shorter. ;-)

```suggestion
; For more information, see:
; https://knowledge.exlibrisgroup.com/Primo/Content_Corner/Central_Discovery_Index/Documentation_and_Training/Documentation_and_Training_(English)/CDI_-_The_Central_Discovery_Index/CDI_Document_Attributes
```